.azure-pipelines/gpu-tests.yml

```diff
@@ -19,7 +19,7 @@ pr:
 jobs:
   - job: pytest
     # how long to run the job before automatically cancelling
-    timeoutInMinutes: "45"
+    timeoutInMinutes: "70"
```

Comment:
I strongly suggest not increasing the timeout time considerably. If the job is taking too long because newest torch is slower, then just increase it the number of minutes it requires.

Otherwise when jobs actually hang, the GPU queue will get full